pages/nfttest/mint/_iscnId.vue

```diff
@@ -105,73 +207,81 @@ export default class NFTTestMintPage extends Vue {
         params: {
           iscn_id: this.iscnId,
         },
-        paramsSerializer: params => qs.stringify(params),
-      });
-      this.classId = data.classId;
-      this.apiData = data;
+        paramsSerializer: (params) => qs.stringify(params),
+      })
+      this.classId = data.classId
+      this.apiData = data
     } catch (err) {
-      console.error(err);
+      console.error(err)
     }
   }
 
   async postMintInfo() {
-    const { data } = await this.$axios.post(API_LIKER_NFT_MINT, {
-    }, {
-      params: {
-        iscn_id: this.iscnId,
-        class_id: this.classId,
-      },
-      paramsSerializer: params => qs.stringify(params),
-    });
-    return data;
+    let fdata
+    try {
+      const { data } = await this.$axios.post(
+        API_LIKER_NFT_MINT,
+        {},
+        {
+          params: {
+            iscn_id: this.iscnId,
+            class_id: this.classId,
+          },
+          paramsSerializer: (params) => qs.stringify(params),
+        },
+      )
+      fdata = data
+    } catch (error) {
+      console.error(error)
+    }
```

Comment:
I think throwing the error might allow some simple error display on UI? Or we can also set a global error data to display the error on UI